tempest/src/main/kotlin/app/cash/tempest/internal/DynamoDbQueryable.kt

```diff
@@ -128,4 +142,12 @@ internal class DynamoDbQueryable<K : Any, I : Any>(
     val offsetKey = keyCodec.toApp(offsetKeyAttributes)
     return Offset(offsetKey)
   }
+
+  companion object {
+    // TODO: fill in the rest of the reserved words
+    private val reservedWordsMapping = mapOf(
+        "token" to "#t",
```

Comment:
If we have a list of keywords here, can aliases will just be "#$keyword"?

Reply:
Should we use a more complicated format like `"__$keyword__"` to prevent collision?

Reply:
Now that I'm thinking about this, we could actually deterministically avoid collisions by just prepending "#" until it doesn't collide (which will probably be once, but if somebody has fields named "#token" and "token", then it would still work.

Reply:
I've updated it to do this.